benchexec/tools/nitwit.py

```diff
@@ -48,30 +47,27 @@ def version(self, executable):
     def name(self):
         return "Nitwit"
 
-    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
-        if propertyfile:
-            options = options + ["-p", propertyfile]
-        return [executable] + options + tasks
+    def cmdline(self, executable, options, task, rlimits):
+        if task.property_file:
+            options = options + ["-p", task.property_file]
+        return [executable] + options + list(task.input_files_or_identifier)
```

Comment:
Does nitwit support tasks without an input file? Does it support tasks with more than one input file?

Depending on the answer, I would suggest to use `task.input_files` or `task.single_input_file`. This immediately fails in unsupported use cases with a nice message.

Reply:
Thanks for the clarification, only single input files are supported. Fixed.